wordstats/cognate_files_path.py

```diff
@@ -0,0 +1,67 @@
+
+from .config import DATA_FOLDER_COGNATES, WHITELIST, BLACKLIST, RULES, CANDIDATES
+import os
+import codecs
+
+# useful methods for handling files
+
+def path_of_cognate_languages(primary, secundary):
+    package_directory = os.path.dirname(os.path.abspath(__file__))
+    file_path = package_directory + os.sep + "{0}{1}{2}{3}".format(DATA_FOLDER_COGNATES, os.sep, primary, secundary)
+    return file_path
+
+def path_of_cognate_candidates(primary, secundary, method_name):
+
+    return path_to_cognate_file(primary, secundary, CANDIDATES, method_name)
+
+
+def path_of_cognate_blacklist(primary, secundary, author:str = ""):
+
+    return path_to_cognate_file(primary, secundary,
+                                BLACKLIST if len(author) == 0 else (BLACKLIST + "_" + author))
+
+
+def path_of_cognate_whitelist(primary, secundary, author:str = ""):
+
+    return path_to_cognate_file(primary, secundary,
+                                WHITELIST if len(author) == 0 else (WHITELIST + "_" + author))
+
+
+def path_of_cognate_rules(primary, secundary, method_name, author:str = ""):
+
+    return path_to_cognate_file(primary, secundary, RULES if len(author) == 0 else (RULES + "_" + author)
+                                , method_name)
+
+def path_to_cognate_file(primary, secundary, file_name, method_name = ""):
```

Comment:
this method is private to this file (or module in Python). 
the convention is to prefix it's name with an underscore: `_ path_to_cognate_file`.